podpac/core/coordinates/uniform_coordinates1d.py

```diff
@@ -215,7 +216,10 @@ def from_definition(cls, d):
     def __getitem__(self, index):
         # fallback for non-slices
         if not isinstance(index, slice):
-            return ArrayCoordinates1d(self.coordinates, **self.properties)[index]
+            # The following 3 lines is copied from ArrayCoordinates1d.__getitem__
```

Comment:
These lines were added so that I could cache the `self.coordinates` call. Otherwise these were constantly being recreated.